app/views/admin/editions/_change_notes.html.erb

```diff
@@ -1,24 +1,44 @@
 <fieldset class="js-change-notes">
-  <h3 class="remove-top-margin">What sort of change are you making?</h3>
-  
-  <div class="radio">
-    <%= form.label :minor_change_true do %>
-      <%= form.radio_button(:minor_change, true) %>
-      <strong>Fixing a typo or broken link, a style change or similar</strong>
-      <p class="hint">Users signed up to email alerts won't be notified</p>
+  <h3 class="remove-top-margin">Do users need to know the content has changed?</h3>
+  <p class="hint">Choose yes to notify users of the change. Telling users when published information changes is important for transparency</p>
+
+  <div class="alert alert-info">
+    <h4> Attachments you've added, changed or deleted </h4>
+    <ul>
+    <% edition.attachments.each do |attachment| %>
+      <% if attachment.created_at > (edition.created_at + 5.seconds) %>
+        <li><%= "#{attachment.title} has been created" %> <span class="label label-success">Created</span></li>
+      <% end %>
+      <% if attachment.updated_at > attachment.created_at || (attachment.respond_to?(:govspeak_content) && attachment.govspeak_content.updated_at > attachment.govspeak_content.created_at) %>
+        <li><%= "#{attachment.title} has been updated" %> <span class="label label-warning">Updated</span></li>
+      <% end %>
```

Comment:
It feels like these two predicates should be moved to the model and have their own predicate methods (e.g. def created? and def updated?)